lifecycle/lifecycle/auth/subject.py

```diff
@@ -135,6 +136,9 @@ def _get_subject_name_from_auth_subject(auth_subject: models.AuthSubject) -> str
     else:
         raise ValueError("Unknown auth_subject type")
 
+def regenerate_specific_user_token(user: User):
+    regenerate_auth_token(user)
```

Comment:
What you're actually passing to this function is `auth_subject: models.AuthSubject`. (Auth Subject can be either User, Job or External Consumer)
Also `regenerate_auth_token` expects auth_subject, and that's ok, so just the type annotations are wrong.

```suggestion
def regenerate_specific_user_token(auth_subject: models.AuthSubject):
    regenerate_auth_token(auth_subject)
```